### PR TITLE
jdbc

### PR DESCRIPTION
**Pull request by StakanSoka**

### CODE REVIEW

File: src/main/java/lesson24/queries/req2.png
Comment (DHrytsuk) [inline src/main/java/lesson24/queries/req2.png:1]: This query could be done without subselect

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: but otherwise i'll need to join another table...

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: image

Comment (DHrytsuk) [reply src/main/java/lesson24/queries/req2.png:1]: Yep
and this is correct and should work a bit faster

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: image

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: image

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: image

Comment (StakanSoka) [reply src/main/java/lesson24/queries/req2.png:1]: image

**Pull request merged**